app/src/schemas.ts

```diff
@@ -0,0 +1,4 @@
+import z from 'zod'
+
+// mainnet, goerli, sepolia, optimism
+export const supportedChainIds = z.enum(['1', '5', '11155111', '10'])
```

Comment:
As your `getChainConfig` will throw error for invalid key, you can reuse it to check supported chains (instead of keeping this list 3 times)